ogr/services/github/flag.py

```diff
@@ -64,9 +64,13 @@ def set(
         return GithubCommitFlag(project=project, raw_commit_flag=status, commit=commit)
 
     @property
-    def created(self) -> datetime.datetime:
-        return self._raw_commit_flag.created_at
+    def created(self) -> Optional[datetime.datetime]:
+        if self._raw_commit_flag:
```

Comment:
I have a weird feeling that this shouldn't happen :thinking: and in case it happens, we should be „angry“

Reply:
Hm, good point, let's raise an exception instead of `None` in case `_raw_commit_flag` is not set.